demo/src/lt.tooltip.js

```diff
@@ -64,13 +64,42 @@ angular.module('lt.tooltip', [])
         }
     };
 
+    var positionArrow = function($arrow, direction) {
+        var arrowPlacement;
+
+        switch (direction) {
+            case 'right':
+                arrowPlacement = 'left';
+                break;
+            case 'left':
+                arrowPlacement = 'right';
+                break;
+            case 'bottom':
+                arrowPlacement = 'top'
+                break;
+            case 'top':
+            default:
+                arrowPlacement = 'bottom';
+        }
+
+        $arrow.addClass(arrowPlacement);
+    }
+
     return {
         restrict: 'AE',
         transclude: true,
-        template: '<div class="tooltip"></div><span class="transcluded" ng-transclude></span>',
+        template: function(tElement, tAttrs) {
+            return '<div class="tooltip">' +
+                        '<div class="tooltip-content"></div>' +
+                        '<div class="tooltip-arrow"></div>' +
+                    '</div>' +
+                    '<span class="transcluded" ng-transclude></span>';
+        },
         link: function($scope, $el, $attrs) {
             var el = $el[0],
                 $tooltipEl = angular.element(el.querySelector('.tooltip')),
+                $content = angular.element(el.querySelector('.tooltip-content')),
+                $arrow = angular.element(el.querySelector('.tooltip-arrow')),
```

Comment:
@liamitus - should the names be `$tooltipContent` and `$tooltipArrow` or `$content` and `$arrow`?